pkg/adapterManager/parallelManager_test.go

```diff
@@ -55,12 +56,12 @@ func TestExecute(t *testing.T) {
 			{&configpb.Adapter{Name: c.name}, &configpb.Aspect{Kind: c.name}},
 		}
 
-		o, err := m.Execute(context.Background(), cfg, nil, nil)
-		if c.inErr != nil && err == nil {
-			t.Errorf("m.Execute(...) = %v; want err: %v", err, c.inErr)
+		o := m.Execute(context.Background(), cfg, nil, nil)
```

Comment:
Completely agree. I've been holding off on making that change for a while, waiting for things to settle more first.